themes/src/main/resources/theme/keycloak/common/resources/lib/patternfly4/tile.css

```diff
@@ -0,0 +1,100 @@
+.pf-c-tile {
```

Comment:
@Pepo48 Have you tried this on IE? IE doesn't support `var()` function. The basic screens should be available even in IE, as it was discussed [here](https://github.com/keycloak/keycloak/pull/7470#discussion_r503393541). So, there should be, at least, some similar appearance like with PF4.

Reply:
@mabartos (cc: @vmuzikar ) I completely revamped the css styles of the tile component. Now the screen is IE11 compatible:
![obrázok](https://user-images.githubusercontent.com/12138171/97352393-ffc00c00-1892-11eb-94fe-f7e7d5439ef9.png)